tools/sidekick/config.json

```diff
@@ -1 +1,18 @@
-{}
+{
+  "project": "marketing-hub",
```

Comment:
@mdickson-adbe I'm unaware of what this change does; can you explain it for me? Thanks

Reply:
@nithinasokan this was included at @shiv-gup request, it's meant to provide a configuration to use the assets picker feature in the sidekick, so we can use assets from the gmo assets repo when building pages.